### DISCUSSION TITLE
Are external link cue and open in new tab supported for markdown links?

### DISCUSSION THREAD

**Discussion by ckamps** [gethinode/hinode#778] [Help]
Are the external link cue and open in new tab features supported for markdown native link references?

![image](https://github.com/gethinode/hinode/assets/1332810/772e9d1d-b11a-4987-a44c-7a3a106b03e8)

The source for this example:

```
[test](https://google.com)

{{< link fontawesome />}}
```

I'm able to get the external link cue and open in new tab to work for managed links when the `link` shortcode is used, but I've been unable to get the cue and open in new tab features to appear when using markdown link references.

```
[main]
    [main.externalLinks]
        cue = true
        tab = true
```

**Comment by markdumay** [comment]
Currently only managed links support this behavior. I'm not sure if Hugo provides a processing hook for markdown links. An alternative would be to use a client-side script.

**Comment by markdumay** [comment]
Hugo v0.123.0 mentions implementation of [image and render hooks](https://github.com/gohugoio/hugo/issues/11933).  This might be a good starting point to render the visual cue for regular markdown links.
